config/index.js

```diff
@@ -334,6 +334,32 @@ var conf = convict({
       env: 'SUPPORTED_LANGS'
     }
   },
+  useRedis : {
+    default: true,
+    doc: 'Enable redis cache',
+    format: Boolean,
+    env: 'USE_REDIS'
+  },
+  redis: {
```

Comment:
@udaraweerasinghege @philbooth we are going to be using redis for `fxa-sessions` and `fxa-devices` (device list cache). I wonder if we need separate redis host configurations or at least support for multiple key prefixes 

cc @jbuck